pages/instructorQuestions/instructorQuestions.sql

```diff
@@ -34,7 +34,7 @@ FROM
   JOIN topics AS top ON (top.id = q.topic_id)
   LEFT JOIN issue_count ON (issue_count.question_id = q.id)
 WHERE
-  q.course_id = $course_id
+  q.course_id = $course_id -- leave the same for now? eventually we will have a way to view shared questions but not now
```

Comment:
SGTM.

Reply:
Same question, should I leave a TODO comment here in the codebase or no?